docs/pages/includes/server-access/azure-join-system-assigned-identity.mdx

```diff
@@ -0,0 +1,49 @@
+<Tabs>
+<TabItem label="Azure Portal">
+
+To set up a Managed Identity:
+1. Navigate to [Virtual machines view](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines)
+if you're hosting Teleport on an Azure VM,
+or navigate to [Virtual machine scale sets view](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachineScaleSets)
+if you're hosting Teleport on an Azure VMSS.
+2. Select the VM or VMSS hosting your Teleport Service.
+3. In the right-side panel, click the **Security/Identity** tab.
+4. Under the **Identity** section, select the **System assigned** tab.
+5. Toggle the **Status** switch to **On**.
+6. Click **Save**.
+
+<Admonition type="note">
+  If you're using VMSS and it is configured with manual upgrade mode, you must
+  update the VM instances for the identity changes to take effect:
+  - Click the **Instances** tab in the right panel.
+  - Select the VM instances to update.
+  - Click **Restart**.
+</Admonition>
+
+</TabItem>
+<TabItem label="Azure CLI">
+
+To attach a system-assigned identity to a regular VM, run:
+
+```code
+$ az vm identity assign --resource-group <resource-group> --name <vm-name>
+```
+
+To attach a system-assigned identity to an Azure VMSS, run:
+
+```code
+$ az vmss identity assign --resource-group <resource-group> --name <vmss-name>
+```
+
+<Admonition type="note">
+  If you're using VMSS and it is configured with manual upgrade mode, you must
+  update the VM instances for the identity changes to take effect. Run the following
+  command to propagate the identity change:
+
+  ```code
+  $ az vmss update-instances --resource-group <resource-group> --name <vmss-name> --instance-ids *
+  ```
+</Admonition>
```

Comment:
I think we can remove the `Admonition` tags here. The text that it wraps isn't a note that will easily go missed if we don't add the `Admonition`, but rather a full half of the Azure CLI tab item.